index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+<head>
+  <meta name="viewport" content="width=device-width">
+  <meta charset="utf-8">
+  <title>Responsive</title>
+  <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <nav>
+      <img src="svg/circle.svg" alt="Круг">
+      <ul>
+        <li class="main"><a href="#">ПОИСК</a></li>
+        <li class="main2"><a href="#">МЕНЮ</a></li>
+        <li class="get"><a  href="#">СТОЙМОСТЬ</a></li>
+        <li class="get"><a  href="#">О ПРОЕКТЕ</a></li>
+        <li class="get"><a  href="#">ВОЙТИ</a></li>
+      </ul>
+    </nav>
+  <section>
+    <div class="container1">
+      <img class="img" src="img/Harry.jpg" alt="Harry">
+    </div>
+    <div class="container2">
```

Comment:
```suggestion
    <div class="book-info">
```

тут тоже могло бы быть осмысленное название класса